tests/test_checker/test_noqa.py

```diff
@@ -292,6 +298,13 @@
     },
 )
 
+#: Number and count of violations that would be raised in 3.13+ section.
+SHOULD_BE_RAISED3_13 = types.MappingProxyType(
+    dict.fromkeys(SHOULD_BE_RAISED, 0) | {
+        'WPS476': 1
+    }
+)
```

Comment:
```suggestion
SHOULD_BE_RAISED3_13 = types.MappingProxyType(
    {
        'WPS476': 1
    }
)
```

Let's simplify this case.

Reply:
Testcases check that every violation code is defined in such dictionary, so I was forced to add that line. Formatter ate my comment on this, btw

Reply:
Check out https://github.com/wemake-services/wemake-python-styleguide/blob/0.19.2/tests/test_checker/test_noqa.py#L43